ofproto/ofproto-dpif-xlate.c

```diff
@@ -3148,20 +3244,21 @@ xlate_normal(struct xlate_ctx *ctx)
     }
 
 #if defined(P4OVS)
-    /* Dynamic MAC is learnt, program P4 forwarding table */
-    struct xport *ovs_port = get_ofp_port(in_xbundle->xbridge,
-                                          flow->in_port.ofp_port);
-    struct mac_learning_info fdb_info;
-    memset(&fdb_info, 0, sizeof(fdb_info));
-
-    if (!get_fdb_data(ovs_port, flow->dl_src, &fdb_info)) {
-        ConfigFdbTableEntry(fdb_info, true);
-    } else {
-        VLOG_ERR("Error retrieving FDB information, skipping programming "
-                 "P4 entry");
-    }
+        /* Dynamic MAC is learnt, program P4 forwarding table */
```

Comment:
Indentation does not match surrounding code.

Reply:
Done